internal/servicedeployer/custom_agent.go

```diff
@@ -99,7 +100,7 @@ func (d *CustomAgentDeployer) SetUp(inCtxt ServiceContext) (DeployedService, err
 		},
 	}
 
-	outCtxt := inCtxt
+	outCtxt := svcInfo
```

Comment:
Following the `svcInfo`naming, should `outCtxtp  be renamed to `outSvcInfo` or similar ?

Reply:
Actually I wonder if this output variable is needed, `svcInfo` is already passed by value, I guess we don't need to copy it again.

I am removing this line in https://github.com/elastic/elastic-package/pull/1679/commits/f784e23ca85095ada3ff72953e5057479dee7cb6